WNPRC_Virology/src/org/labkey/wnprc_virology/WNPRC_VirologyModule.java

```diff
@@ -0,0 +1,126 @@
+package org.labkey.wnprc_virology;
+
+import org.jetbrains.annotations.NotNull;
+import org.labkey.api.data.Container;
+import org.labkey.api.data.ContainerManager;
+import org.labkey.api.ehr.EHRService;
+import org.labkey.api.ldk.notification.Notification;
+import org.labkey.api.ldk.notification.NotificationService;
+import org.labkey.api.module.FolderTypeManager;
+import org.labkey.api.module.Module;
+import org.labkey.api.module.ModuleContext;
+import org.labkey.api.module.SpringModule;
+import org.labkey.api.query.DefaultSchema;
+import org.labkey.api.query.QuerySchema;
+import org.labkey.api.security.roles.RoleManager;
+import org.labkey.api.view.WebPartFactory;
+import org.labkey.api.view.template.ClientDependency;
+import org.labkey.wnprc_virology.notification.ViralLoadQueueNotification;
+import org.labkey.wnprc_virology.notification.ViralLoadQueueNotificationSummaryEmail;
+import org.labkey.wnprc_virology.security.roles.WNPRCViralLoadReaderRole;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+public class WNPRC_VirologyModule extends SpringModule
+{
+    public static final String NAME = "WNPRC_Virology";
+
+    public static final String VIROLOGY_EHR_VL_SAMPLE_QUEUE_PATH_PROP = "virologyEHRVLSampleQueueFolderPath";
+    public static final String RSEHR_QC_STATUS_STRING_PROP = "RSEHRQCStatus";
+    public static final String RSEHR_PORTAL_URL_PROP = "RSEHRPortalPath";
+    public static final String RSEHR_PARENT_FOLDER_STRING_PROP = "RSEHRViralLoadDataFolder";
+    public static final String RSEHR_JOB_INTERVAL_PROP = "RSEHRJobInterval"; // in minutes
+    public static final String ZIKA_PORTAL_QC_STATUS_STRING_PROP = "ZikaPortalQCStatus";
+    public static final String ZIKA_PORTAL_URL_PROP = "ZikaPortalPath";
+    //TODO: assemble notify list from ETL from RSEHR
+    public static final String USE_RSEHR_FOLDER_PERMS_FOR_EMAIL = "virologyRSHEREmailMethod";
+
+    public static final String RSEHR_EMAIL_CONTACT_INFO = "RSEHRNotificationEmailReplyTo";
+
+    @Override
+    public String getName()
+    {
+        return NAME;
+    }
+
+    @Override
+    public Double getSchemaVersion()
+    {
+        return 21.111;
```

Comment:
should this be 22.11?

Reply:
to keep it consistent with the core version, I can change it to 22.110